### PR TITLE
903554 - UG Documentation for SfTreeMap

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
Added ug documentation for drilldown and right to left flow direction in Sftreemap

### CODE REVIEW

File: MAUI/TreeMap/drilldown.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
+description: Learn here all about Drilldown feature and the header customization of Syncfusion .NET MAUI TreeMap (SfTreeMap) control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Drilldown in .NET MAUI TreeMap (SfTreeMap)
+
+The drilldown feature provides better visualization of hierarchy. A large set of data can be virtualized into minimal views. Each level of items can be drilled down. TreeMap provides animation along with the drilldown support. The drilldown header will be enabled when you perform drilldown in TreeMap, and this header helps in performing zoom out operation (i.e., drill up one level).
```
Comment (saiganesh-sakthivel) [inline MAUI/TreeMap/drilldown.md:12]: The drilldown feature provides better visualization of hierarchy. A large set of data can be virtualized into minimal views. Each level of items can be drilled down. TreeMap provides animation along with the drilldown support.

File: MAUI/TreeMap/legend.md [hunk 2]
```diff
@@ -617,6 +617,145 @@ N> This property will only be applicable when the [ShowLegend](https://help.sync
...
+ 
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Items Template
+
+The `TreeMapLegend` supports customizing the appearance of legend items using the ItemTemplate property. The default value of ItemTemplate is null.
+
```
Comment (saiganesh-sakthivel) [inline MAUI/TreeMap/legend.md:699]: add the note for the binding context of the itemtempate
refer in API documentation or source example code remarks

File: MAUI/TreeMap/right-to-left.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
+description: Learn here all about the flow direction feature of Syncfusion .NET MAUI TreeMap (SfTreeMap) control.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Right To Left in .NET MAUI TreeMap (SfTreeMap)
+
+SfTreeMap supports changing the flow to the right-to-left direction by setting the FlowDirection to RightToLeft. TreeMap supports RTL when the flow direction of the device is changed.
```
Comment (saiganesh-sakthivel) [inline MAUI/TreeMap/right-to-left.md:12]: SfTreeMap supports changing the flow to the right-to-left direction by setting the FlowDirection to RightToLeft.

File: MAUI/TreeMap/drilldown.md [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+
+SfTreeMap treeMap = new SfTreeMap();
+treeMap.EnableDrilldown = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+![enable-drilldown-in-maui-tree-map](images\drilldown\maui-drilldown.png){:width="313" height="480"  loading="lazy" .lazy .shadow-effect .section-padding .img-padding}
+
+## Header customization
```
[the remark this answers is not in the document]
Comment (SriRadheshNagS) [reply MAUI/TreeMap/drilldown.md:34]: Updated the changes

File: MAUI/TreeMap/drilldown.md (hunk 1 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/drilldown.md:12]: Updated the changes

File: MAUI/TreeMap/legend.md (hunk 2 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/legend.md:699]: Updated thee changes

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

File: MAUI/TreeMap/drilldown.md [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Drilldown in .NET MAUI TreeMap (SfTreeMap)
+
+The drilldown feature provides better visualization of hierarchy. A large set of data can be virtualized into minimal views. Each level of items can be drilled down. TreeMap provides animation along with the drilldown support.
+
+## Enable Drilldown
```
Comment (Jeyasri-Murugan) [inline MAUI/TreeMap/drilldown.md:14]: Enable Drilldown 

Change as Enable drilldown

File: MAUI/TreeMap/legend.md [hunk 6]
```diff
@@ -617,6 +617,147 @@ N> This property will only be applicable when the [ShowLegend](https://help.sync
 
  ![legend-size-in-maui-tree-map](images/legend/legend-size-in-maui-tree-map.png)
 
+
+## Items Layout
```
Comment (Jeyasri-Murugan) [inline MAUI/TreeMap/legend.md:621]: Items Layout - Change as Customize items layout

File: MAUI/TreeMap/legend.md [hunk 7]
```diff
@@ -617,6 +617,147 @@ N> This property will only be applicable when the [ShowLegend](https://help.sync
...
+        new TreeMapRangeBrush { LegendLabel = "0.1M - 10M", From = 100000, To = 10000000, Brush = new SolidColorBrush(Color.FromArgb("#7DC59D")) },
+    }
+};
+ 
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Items Template
```
Comment (Jeyasri-Murugan) [inline MAUI/TreeMap/legend.md:696]: Items Template - change as Customize items template

File: MAUI/TreeMap/right-to-left.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/TreeMap/right-to-left.md:3]: Add proper title - title: Drilldown in .NET MAUI TreeMap Control | Syncfusion

File: maui-toc.html [hunk 9]
```diff
@@ -1255,6 +1255,8 @@
             <li><a href="/maui/TreeMap/selection-and-highlights">Selection and Highlight</a></li>
             <li><a href="/maui/TreeMap/events">Events</a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toc.html:1256]: Change **Drilldown** order next to **Levels**
Change **RTL** order before to **Events**

File: MAUI/TreeMap/right-to-left.md (hunk 3 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/right-to-left.md]: Updated the changes na

File: maui-toc.html (hunk 9 above)
Comment (SriRadheshNagS) [reply maui-toc.html:1257]: Updated changes

File: maui-toc.html [hunk 10]
```diff
@@ -1255,6 +1255,8 @@
...
             <li><a href="/maui/TreeMap/accessibility">Accessibility</a></li>
+            <li><a href="/maui/TreeMap/drilldown">Drilldown</a></li>
+            <li><a href="/maui/TreeMap/right-to-left">Right to left</a></li>
         </ul>
```
[the remark this answers is not in the document]
Comment (SriRadheshNagS) [reply maui-toc.html:1260]: updated changes

File: MAUI/TreeMap/right-to-left.md (hunk 8 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/right-to-left.md:3]: changes updated

File: MAUI/TreeMap/legend.md (hunk 7 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/legend.md]: updated the changes

File: MAUI/TreeMap/legend.md (hunk 6 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/legend.md]: updated the changes

File: MAUI/TreeMap/drilldown.md (hunk 5 above)
Comment (SriRadheshNagS) [reply MAUI/TreeMap/drilldown.md]: updated the changes

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**